BUILDING.md

```diff
@@ -105,7 +105,11 @@ You need to install the following
 - Xcode
 - [Doxygen](#doxygen) (only if generating documentation)
 
-For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk).
+For the load tests applications you need to install the Vulkan SDK.
+To build for macOS you need set the CMake cache variable `VULKAN_SDK` to the root of the Vulkan SDK, if it is not already set.
```

Comment:
Delete this line. It is no longer necessary.

Reply:
Oh! I missed those 2. It is unfortunate that find_package(Vulkan) doesn't provide more information. The ICD is installed in /usr/local/share/ so could be found by deriving VULKAN_SDK from Vulkan_LIBRARIES as I did for Vulkan_LIBRARY_DIR. However the framework does not appear to be installed anywhere but inside the SDK. I will experiment with copying libvulkan.dylib to the app bundles instead.

Reply:
`vulkan.framework` is not needed in the app bundle because the app is linked to the .dylib which is copied to the bundle by one of the lines you modified yesterday. So please delete lines 260 to 263 of vkloadtests.cmake.

VK_ICD is not used so you can also delete line 194. Since the libraries are in the app bundle we need a custom icd.json which is stored in other_lib/mac/resources and which is set in the variable MOLTEN_VK_ICD and is copied to the bundle. We do not need the icd.json from the SDK.

I have verified these changes. After them, the app still works both when run in Xcode and when run standalone.

Indeed the macOS build does not need VULKAN_SDK, except possibly to help cmake find the package.